docs/src/main/sphinx/installation/cli.rst

```diff
@@ -405,6 +415,24 @@ Press :kbd:`→` to accept the suggestion and replace the current command line
 buffer. Press :kbd:`Ctrl+→` (:kbd:`Option+→` on Mac) to accept only the next
 keyword. Continue typing to reject the suggestion.
 
+Configuration file
```

Comment:
Thanks for documenting this feature that I've added a long time ago :)

Reply:
You can return the favor by reviewing this PR :D